frontend/src/components/CampaignPlanning.vue

```diff
@@ -241,7 +247,21 @@ export default {
     this.fetchCampaign(campaignName);
   },
   methods: {
-    addEntry: function() {
+    addEntryAction: function() {
+      console.log('addEntryAction');
```

Comment:
Delete all `console.log`. They do not hurt, but build script is unhappy about them